src/abandoned-cart-email-widget/abandoned-cart-email-widget.html

```diff
@@ -0,0 +1,16 @@
+
+<!-- 
+
+The Sezzle Shopify abandoned cart email widget is a lightweight version of our static widget.It suggests the merchants to use Sezzle widget in their brand’s email footers and abandoned cart emails.
+It is mainly self-contained widget version for implementation without connection to Sezzle’s server, which can be copied to the merchant theme via Github
+
+Log into Shopify Admin
+Go to Settings > Notifications > Abandoned Checkout
+Click the <> (Show HTML) button within the description or wthin footer tag
+Paste the static widget at the top or bottom of the code window, depending on where you would like the widget to appear
```

Comment:
This should probably be edited so it reads "Paste the below code snippet where you would like the widget to appear." Probably also a good idea to include a suggestion, such as between the products table and the "row footer" table.

Reply:
fixed